integrations/adding-an-integration/conneceting-to-k8s-cluster.md

```diff
@@ -0,0 +1,23 @@
+# Connecting to K8s cluster
+
+Connecting Aqueduct to a K8s cluster requires a `kubeconfig` file that allows a kubectl client to connect to the cluster.&#x20;
```

Comment:
what's the ampersand code here?

Reply:
We should specify that this file must be present on the same machine as the Aqueduct server.